source/guide/index.md

```diff
@@ -10,124 +10,125 @@ version: 2.1
 
-Under the scene, Weex uses the native widgets. Although Weex emphasize the consistency on each mobile platform, we still embrace the platform's own behavior and UI differences. For example, [the `<switch>` component](http://dotwe.org/vue/d96943452b6708422197c47920903823) may look different on Android and iOS (the appearance on the web simulates iOS).
+Behind the scenes, Weex uses native widgets. Although Weex emphasizes consistency on each mobile platform, we still embrace the platform's own behavior and UI differences. For example, [the `<switch>` component](http://dotwe.org/vue/d96943452b6708422197c47920903823) may look different on Android and iOS (the appearance on the web simulates iOS).
 
 ![Different switch](./images/different-switch.png)
 
-If you want to use more native component other than the built-in components, you need to implement them on each platform and keep their behaviors to be consistent. The most practical way is to integrate the existing native components to Weex platform.
+If you want to use additional native components, other than the built-in components Weex provides, you need to implement them on each platform and keep their behaviors consistent. The most practical way is to integrate the existing native components to Weex platform. /_ need explanation _/
 
 ### Native Modules
 
-For those features which does not rely on the UI, Weex wraps them into **modules**. You can use `weex.requireModule('xxx')` to require them. It is a way to access native capabilities in javascript, such as network, storage, clipboard, and navigator. For example, you can [use `stream` module to fetch the star count of Vue.js](http://dotwe.org/vue/2ae062b6a04124a35bbe2da3b1e5c07b).
+For those features that do not rely on the UI, Weex wraps them into **modules**. You can use `weex.requireModule('xxx')` to require them. Weex modules provide easy access to native capabilities in JavaScript, such as network, storage, clipboard, and navigator. For example, you can [use `stream` module to fetch the star count of Vue.js](http://dotwe.org/vue/2ae062b6a04124a35bbe2da3b1e5c07b).
 
-Similarly, Weex also provides a group of [built-in modules](../references/modules/) for basic usage, and supports the integration of the existing native modules to Weex platform.
+Similarly, Weex provides a group of [built-in modules](../references/modules/) for basic usage, and supports the integration of the existing native modules into the Weex platform.
 
 Here are some documents about how to extend native components and native modules for Weex:
 
-+ [Extend Web Render](./extend-web-render.html)
-+ [Extend Android](./extend-android.html)
-+ [Extend iOS](./extend-ios.html)
+- [Extend Web Render](./extend-web-render.html)
+- [Extend Android](./extend-android.html)
+- [Extend iOS](./extend-ios.html)
 
 ### Write Once, Run Everywhere
 
-Yes, Weex can build for Android, iOS and Web from a single codebase.
+Yes, Weex can build for Android, iOS, and Web from a single codebase.
```

Comment:
Should we mention "...and Web apps..." instead of "...and Web..."? I've observed a few people confusing this with website instead of a mobile site.